check_api/src/main/java/com/google/errorprone/ErrorProneAnalyzer.java

```diff
@@ -57,6 +61,73 @@ public class ErrorProneAnalyzer implements TaskListener {
   private final Context context;
   private final DescriptionListener.Factory descriptionListenerFactory;
 
+  public static ErrorProneAnalyzer createAnalyzer(
+      ScannerSupplier scannerSupplier,
+      ErrorProneOptions epOptions,
+      Context context,
+      RefactoringCollection[] refactoringCollection) {
+    if (!epOptions.patchingOptions().doRefactor()) {
+      return createByScanningForPlugins(scannerSupplier, epOptions, context);
+    }
+    refactoringCollection[0] = RefactoringCollection.refactor(epOptions.patchingOptions(), context);
+
+    // Refaster refactorer or using builtin checks
+    CodeTransformer codeTransformer =
+        epOptions
+            .patchingOptions()
+            .customRefactorer()
+            .or(
+                () -> {
+                  ScannerSupplier toUse =
```

Comment:
This un-does some changes from https://github.com/google/error-prone/commit/9da2d5580e3939f97ef2e91278b330a56b5ed1fe compared to the original code, I wonder if that was just an oversight when picking up 9da2d5580e3939f97ef2e91278b330a56b5ed1fe?

Reply:
Yes, was an oversight, re-pushed the changes using the latest code.